server/api/openapi.yaml

```diff
@@ -62,6 +62,36 @@ paths:
       summary: Get a list of APIs
       tags:
       - apis
+    post:
+      description: Deploys a new API to the cluster
+      operationId: deployApi
+      requestBody:
+        $ref: '#/components/requestBodies/inline_object'
+        content:
```

Comment:
remove the content if you have a ref?

Reply:
It seems this file gets auto generated and we can ignore it
We are using the one in the root of the project